docker-compose.yml

```diff
@@ -54,6 +54,10 @@ services:
       - "traefik.http.routers.traefik-dash.entrypoints=http,https"
       - "traefik.http.routers.traefik-dash.rule=HostRegexp(`traefik{domain:.+}`)"
       - "traefik.http.routers.traefik-dash.service=dashboard@internal"
+      - "traefik.http.routers.domains.tls=true"
+      - "traefik.http.routers.domains.tls.certresolver=le"
+      - "traefik.http.routers.domains.tls.domains[0].main=traefik-kitt.${KITT_DOMAIN}"
```

Comment:
i think i missed the conversations where the traefik host is named traefik-kitt instead of just traefik? the cloudflare argo tunnel is named traefik (w/o -kitt) in my configuration.

Reply:
The tunnel is named traefik (has not changed).  The traefik-kitt is needed since we aren't using kitt.run anymore.  Since it is on your domain, it has to be different from traefik.

It is expected that traefik won't create the specific certs -- it is there if you opt to not use wildcards.

Reply:
The private `traefik-kitt` is used between CF and Traefik, uses a public LE cert.  The `traefik` is a public endpoint created by CF (along with a CF generated cert), used to CNAME multiple names over the same CF Argo tunnel.